src/components/SearchResults.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import moment from "moment";
+
+export default class SearchResults extends Component {
+  constructor() {
+    super();
+    this.state = {};
```

Comment:
This initial state should have a `selected` property, if you're going to say the following later:
```return { selected: !pre.selected };```

However, I'm not sure that having all of SearchResults have a selected flag makes any sense - we want to track selection of specific rows.